pkg/operator/ceph/cluster/osd/topology_test.go

```diff
@@ -41,12 +42,12 @@ func TestOrderedCRUSHLabels(t *testing.T) {
 func TestTopologyLabels(t *testing.T) {
 	// load all the expected labels
 	nodeLabels := map[string]string{
-		"topology.kubernetes.io/region": "r.region",
-		"topology.kubernetes.io/zone":   "z.zone",
-		"kubernetes.io/hostname":        "host.name",
-		"topology.rook.io/rack":         "r.rack",
-		"topology.rook.io/row":          "r.row",
-		"topology.rook.io/datacenter":   "d.datacenter",
+		corev1.LabelZoneRegionStable:        "r.region",
```

Comment:
Available in all our supported k8s versions?

Reply:
Yes, we only support these labels. I forget in which version of K8s these topology labels were added, but they have been around a while.